docs/markdown/Gnome-module.md

```diff
@@ -93,6 +93,7 @@ There are several keyword arguments. Many of these map directly to the
 `g-ir-scanner` tool so see its documentation for more information.
 
 * `dependencies`: deps to use during introspection scanning
+* `doc_format`: *(Added 1.7.1)* format of the inline documentation
```

Comment:
1.8.0, because new features do not get backported to "regression fixes" maintenance branches. :P